src/components/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './UserInfo.scss';
+
+import { Users } from '../../types/Users';
+
+type Props = {
+  user: Users | undefined;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <div className="user-info">
+    <h2 className="user-info__name">
+      {user?.name}
```

Comment:
```suggestion
      {user.name}
```
Якщо прибрати з типізації undefined, то можна позбутись цього знака запитання